test/utilities.ts

```diff
@@ -0,0 +1,53 @@
+import Web3 from "web3";
+import {HttpProvider} from "web3-core";
+import {EpochTokenLockerInstance} from "../types/truffle-typings";
+
+const jsonrpc = "2.0";
+const id = 0;
+const send = function (method: string, params: any, web3Provider: Web3) {
+  return new Promise(function (resolve, reject) {
+    (web3Provider.currentProvider as HttpProvider).send(
+      {id, jsonrpc, method, params},
+      (error, result) => {
+        if (error) {
+          reject(error);
+        } else {
+          resolve(result);
+        }
+      }
+    );
+  });
+};
+
+// Wait for n blocks to pass
+/**
+ * Wait for n (evm) seconds to pass
+ * @param seconds: int
+ * @param web3Provider: potentially different in contract tests and system end-to-end testing.
+ */
+export const waitForNSeconds = async function (
+  seconds: number,
+  web3Provider = (web3 as any) as Web3
+) {
+  await send("evm_increaseTime", [seconds], web3Provider);
+  await send("evm_mine", [], web3Provider);
+};
+
+export const closeAuction = async (
```

Comment:
Is this method is duplicated?

Reply:
If so, this may be due to the fact that we had two different types of auctions.... snapp and stablex

Reply:
What I mean is that it appears to have been added to `utilities.ts` but did not exist in `utilities.js` and seems to exist somewhere else as well.

Reply:
It's duplicated from scripts. I haven't really decided on the final structure yet, but ideally test and script should not depend on one another (maybe on some base utilities but I want to avoid cross references) so they can be separate packages.